subt/ros/robot/launch/freyja.launch

```diff
@@ -174,9 +174,11 @@
   <node pkg="topic_tools" type="relay" name="points_merge_front" args="$(arg robot_name)/rgbd_front/points $(arg robot_name)/rgbd_all/points"/>
   <node pkg="topic_tools" type="relay" name="points_merge_rear" args="$(arg robot_name)/rgbd_rear/points $(arg robot_name)/rgbd_all/points"/>
 
-  <node if="0" ns="mapping" pkg="octomap_server" type="octomap_server_node" name="octomap">
+  <node if="$(eval arg('robot_name').endswith('XM'))" ns="mapping" pkg="octomap_server" type="octomap_server_node" name="octomap">
```

Comment:
For my education: What is the limit on robot name length?

Reply:
_Name can't have more than 24 characters_ ... if I try to put there long string